keyboards/idank/sweep/keymaps/default/keymap.json

```diff
@@ -0,0 +1,106 @@
+{    "version": 1,
+    "notes": "My awesome keymap",
+    "documentation": "\"This file is a QMK Configurator export. You can import this at <https://config.qmk.fm>. It can also be used directly with QMK's source code.\n\nTo setup your QMK environment check out the tutorial: <https://docs.qmk.fm/#/newbs>\n\nYou can convert this file to a keymap.c using this command: `qmk json2c {keymap}`\n\nYou can compile this keymap using this command: `qmk compile {keymap}`\"\n",
+    "keyboard": "ferris/0_1",
+    "keymap": "default",
+    "layout": "LAYOUT_split_3x5_2",
```

Comment:
Please edit this file appropriately for the keyboard you are adding, rather than blindly copy pasting from existing keyboards.